backend/tests/api/test_evaluation.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from bson.objectid import ObjectId
+
+
+@pytest.fixture
+def insert_example_entry(db):
+    entry_doc = {
+        "firstName": "Sebastian",
+        "lastName": "Wiadro",
+        "guardianFirstName": "Sebastian",
+        "guardianLastName": "Senior",
+    }
+
+    id = db["entries"].insert_one(entry_doc.copy()).inserted_id
+    return id
+
+
+def test_evaluation(client, insert_example_entry: ObjectId):
+    id = insert_example_entry
+    evaluation = [{"value": "laureat", "entryId": str(id)}]
+    response = client.post("/evaluation/", json=evaluation)
+
+    assert response.status_code == 200
+    assert response.json() == {"modifiedCount": 1}
```

Comment:
You should get the entry using entries endpoint and see if it really changed it. No shortcuts silly… 
![IMG_0374](https://github.com/kozlowskimaciej/because-its-worth-contest-platform/assets/67715928/ed81c008-1130-45be-af0e-d4f651d6f7db)

Reply:
Or fetch it directly using db fixture